api/dimension_info.md

```diff
@@ -1,41 +1,36 @@
 # Dimension Builders
 
-<Highlight type="note">
-Dimension builders were introduced in TimescaleDB 2.13.
-</Highlight>
-
-The `create_hypertable` and `add_dimension` are used together with
-dimension builders to specify the dimensions to partition a
-hypertable on.
+You call [`create_hypertable`][create_hypertable] and [`add_dimension`][add_dimension] to specify the dimensions to 
+partition a hypertable on. TimescaleDB supports partitioning [`by_range`][by-range] and [`by_hash`][by-hash]. You can 
+partition `by_range` on it's own. However, because `by_hash` makes a fixed number of partitions which can grow 
+very large, best practice is to set `by_range` when you use `by_hash`. 
```

Comment:
Here I suggest we say that hypertables must always have a primary range/time dimension, followed by an arbitrary number of additional dimensions that can be either range or hash, but typically just one hash.

Reply:
Done